Makefile

```diff
@@ -0,0 +1,11 @@
+project = lisp
+
+all:run
+
+# If dependancies are not installed attempt to install them using
+# the qlot executable
+.qlot/setup.lisp:
+	command -v qlot >/dev/null && qlot install
+
+run: .qlot/setup.lisp
```

Comment:
the `run` target should be marked as phony. if you create a file named `run` the makefile will not run this command if the file `run` is newer than the dependencies.

Just add the line `.PHONY: run`